cdap-hbase-spi/src/main/java/co/cask/cdap/spi/hbase/HBaseDDLExecutor.java

```diff
@@ -55,6 +59,8 @@
 
   /**
    * Create the specified table if it does not exist.
+   * This method is called during the creation of dataset(either system or user)
```

Comment:
This method is called during the creation of an HBase backed dataset (either system or user).